doc/sphinx/source/_templates/empty

```diff
@@ -0,0 +1 @@
+This directory is empty.
```

Comment:
Never trust a file named empty...

I believe chpldoc (sphinx under the hood) generates _templates/empty and _static/empty as place-holders so that git will track the directories . The sphinx Makefile expects these directories so we need the dummy files in them.